### ISSUE TITLE
vague wording in Exercise 5, Intro to Data lab

### ISSUE THREAD

**Issue by beanumber** [OpenIntroStat/oilabs-tidy#47]
> Which month has the highest average departure delay from an NYC airport? What about the highest median departure delay? Which of these measures is more reliable for deciding which month(s) to avoid flying if you really dislike delayed flights.

I'm not sure what you are going for here because of the ambiguity of "really dislike delayed flights". It seems to me that if you are trying to avoid **a single really long delay**, then you should base your judgment on the mean. If you are trying to avoid **any kind of delay**, then you might base your judgment on the median. 

Could this be tightened up?

**Comment by mine-cetinkaya-rundel** [comment]
I was going for students getting into the discussion you bring up here. But we can reword it to be a question that has a single correct answer instead of a discussion.

Or reword it so they discuss "what if you go by the mean? what if you go by the median?" so it's more clear the expected answer is a discussion? I like this a bit better but not sure if it's introducing too much ambiguity.

**Comment by beanumber** [comment]
I see. I like the latter idea. I'm happy with an open-ended question but the current wording just seems vague.

**Comment by mine-cetinkaya-rundel** [comment]
What about 

> "Suppose you really dislike departure delays, and you want to schedule your travel in a month that minimizes your potential departure delay leaving NYC. One option is to choose the month with the lowest mean departure delay. Another option is to choose the month with the lowest median departure delay. What are the pros and cons of these two choices?"

**Comment by beanumber** [comment]
Much better, thanks.
